src/server_manager/electron_app/digitalocean_oauth.ts

```diff
@@ -168,15 +167,15 @@ export function runOauth(): OauthSession {
       const accessToken = params.get('access_token');
       if (accessToken) {
         getAccount(accessToken)
-            .then((account) => {
-              if (account.status === 'active') {
-                response.send(closeWindowHtml('Authentication successful'));
-              } else {
-                response.redirect('https://cloud.digitalocean.com');
-              }
-              resolve(accessToken);
-            })
-            .catch(reject);
+        .then((account) => {
```

Comment:
Is this the autoformatter?
I find it easier to read if you move the `then` call to the previous line. This way the identation is preserved.